functests/xml-tests.yaml

```
@@ -46,6 +46,53 @@ tests:
     expected_output: |
       <unit><number value="99"/></unit>
 
+  - name: "Convert non-finite number to XML #1"
+    command: "{command} --format xml --decimal"
+    input: |
+      ⦰ 
+    expected_output: |
+      <unit>
+        <number value="&#x29B0;" decimal="nan"/>
+      </unit>
+
+  - name: "Convert non-finite number to XML #2"
+    command: "{command} --format xml --decimal"
+    input: |
+      -⦰ 
+    expected_output: |
+      <unit><number value="-&#x29B0;" decimal="nan"/></unit>
+
+  - name: "Convert non-finite number to XML #3"
+    command: "{command} --format xml --decimal"
+    input: |
+      +⦰ 
+    expected_output: |
+      <unit><number value="+⦰" decimal="nan"/></unit>
+
+  - name: "Convert non-finite number to XML #1"
```

This appears to be a duplicate test name, which could cause confusion if it was to fail. Could it be renamed?

```suggestion
  - name: "Convert non-finite number to XML #4"
```